src/models/forms/neuburgForm.js

```diff
@@ -1,5 +1,93 @@
 // @flow
 
 export default {
-  temp: Number
+  // landlord's data
+  surname: {
+    type: String,
+    required: true
+  },
+  firstName: {
+    type: String
+  },
+  street: {
+    type: String
+  },
+  zip: {
+    type: Number,
+    min: 10000,
+    max: 99999
+  },
+  location: {
+    type: String
+  },
+  phone: {
+    type: String
+  },
+
+  // data on the object
+  totalArea: {
+    type: Number,
+    required: true
+  },
+  totalRooms: {
+    type: Number,
+    min: 1,
+    required: true
+  },
+  ofRooms: {
+    type: [String],
+    lowercase: true,
+    enum: ['kitchen', 'bath', 'wc', 'child1', 'child2',
+      'child3', 'bed', 'hallway', 'store', 'basement', 'balcony'],
+    validate: {
+      validator: function (values: Array<string>): boolean {
```

Comment:
We should sync the eslint configs :) this should be prohibited by linting

Reply:
https://integreat.atlassian.net/browse/WEBAPP-274

Reply:
Although I'm not entirely sure, if this would be prevented by our current webapp-linter :D
sometimes you do want to define functions like that, if you don't want to have the `this` binded in the current context (which should be rare - although it's needed with the mongoose-binder I think 🤔 )

Reply:
As @Schedulaar already pointed out, when using lambda functions `this` refers to another context. 
In the case you commented it wouldn't make any difference, but if you have a look at the other validator functions, `this` is used. For consistency I'd like to have "normal" function in all mongoose.Schema objects :)

Reply:
One alternative would be: http://mongoosejs.com/docs/advanced_schemas.html

Or: `(() => this.data).bind(this)`